src/Models/Invoice.php

```
@@ -65,91 +52,36 @@ public function __construct()
-                    $item->quantity($itemData['quantity']);
-                }
-
-                if (isset($itemData['unit_price'])) {
-                    $item->unitPrice($itemData['unit_price']);
-                }
-
-                if (isset($itemData['tax_percentage'])) {
-                    $item->taxPercentage($itemData['tax_percentage']);
-                }
+            $items = array_map(
+                static fn (array $itemData) => InvoiceItem::make($itemData),
```

PHPMD (2.15.0)

Avoid using static access to class 'SimpleParkBv\Invoices\Models\InvoiceItem' in method 'make'. (undefined)

(StaticAccess)